.github/workflows/ci.yml

```diff
@@ -396,7 +396,7 @@ jobs:
             uname -a
           run: |
             export PATH=$HOME/.rust_solaris/bin:$PATH
-            cargo build --target x86_64-pc-solaris --all-targets --all-features && sudo cargo test
+            cargo build --target x86_64-pc-solaris --all-targets --all-features && sudo env PATH=$PATH cargo test
```

Comment:
This looks like exactly what `sudo --preserve-env=PATH` is for.  Does solaris's sudo have that option?

Reply:
> This looks like exactly what `sudo --preserve-env=PATH` is for. Does solaris's sudo have that option?

It does. But I think it needs to be enabled somewhere in configuration. Which doesn't seem to be the case.